src/Xamarin.Android.Build.Tasks/Utilities/ZipArchiveEx.cs

```diff
@@ -17,6 +17,10 @@ public class ZipArchiveEx : IDisposable
 			get { return zip; }
 		}
 
+		public bool AutoFlush { get; set; } = true;
+
+		public bool CreateDirectoriesInZip { get; set; } = true;
```

Comment:
Naming: I'm not sure we need the `InZip` suffix; what else could it refer to?

Also, as discussed earlier, I don't think we need this *at all*: in what circumstances in our toolchain do we need an explicit directory entry in the first place?  We can probably just skip this property and the `zip.CreateDirectory (fullDirPath)` invocation entirely.